packages/amplify-provider-awscloudformation/src/graphql-transformer/utils.ts

```diff
@@ -85,6 +85,20 @@ export function readFromPath(directory: string): any {
   return accum;
 }
 
+export function mergeUserConfigWithTransformOutput(userConfig: any, transformOutput: any) {
+  const userResolvers = userConfig.resolvers || {};
+  const userPipelineFunctions = userConfig.pipelineFunctions || {};
+  const pipelineFunctions = transformOutput.pipelineFunctions;
+
+  for (const userResolver of Object.keys(userResolvers)) pipelineFunctions[userResolver] = userResolvers[userResolver];
+  for (const userResolver of Object.keys(userPipelineFunctions)) pipelineFunctions[userResolver] = userPipelineFunctions[userResolver];
```

Comment:
Can we also include user defined functions, stacks and the necessary stack edits?
(similar to amplifyUtils)
https://github.com/aws-amplify/amplify-cli/blob/b1229477e871a7e4724131bb709f55f5f77358cb/packages/graphql-transformer-core/src/util/amplifyUtils.ts#L252-L254

Reply:
@SwaySway I'm going to create a follow up PR to handle integrating stacks and functions since this is just for overwriting resolvers/pipeline functions.